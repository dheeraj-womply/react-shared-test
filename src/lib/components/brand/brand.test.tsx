import React from 'react'
import { Logo } from './index'
import renderer from 'react-test-renderer'

test('Logo DOM elements are OK', () => {
    const component = renderer.create(<Logo alt="Womply" />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
