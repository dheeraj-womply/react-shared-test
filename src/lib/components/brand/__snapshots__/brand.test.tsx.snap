// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo DOM elements are OK 1`] = `
<div
  className="sc-AxjAm lgwZkj"
>
  <img
    alt="Womply"
    src="https://res.cloudinary.com/womply/image/upload/react-shared-components/womply_vt6xjk.png"
  />
</div>
`;
